src/cljs/rems/administration/create_catalogue_item.cljs

```diff
@@ -112,9 +112,10 @@
   {:get-form ::form
    :update-form ::set-form-field})
 
-(defn- catalogue-item-title-field []
-  [text-field context {:keys [:title]
-                       :label (text :t.create-catalogue-item/title)
+(defn- catalogue-item-title-field [language]
+  [text-field context {:keys [:title language]
+                       :label (str (text :t.create-catalogue-item/title)
```

Comment:
You should use `text-format` here. 

Also notice that these fields now look different to the fields on the other pages such as Create Form or even Catalogue Item Details. It would be nice for there to be one consistent format on how to display localization texts.

Reply:
Created #1334 for the unification.